subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -230,7 +225,7 @@ You can enable the feature in a couple of ways:
 
 Run with `--watch-fs` on the command line::
 This enables watching the file system for this build only.
-Put `org.gradle.unsafe.watch-fs=true` in your `gradle.properties`::
+Put `org.gradle.vfs.watch=true` in your `gradle.properties`::
```

Comment:
Shall we update the blog post for the 6.7 release?

Reply:
I added https://github.com/gradle/gradle-private/issues/3191.